src/EPR.Calculator.Service.Common/Utils/Util.cs

```diff
@@ -28,6 +29,11 @@ public static string GetFinancialYearAsYYYY(string value)
             }
 
             var years = value.Split('-');
+            if (years.Length != 2 || !int.TryParse(years[1], out _))
```

Comment:
Please use a constant for 2. It will be intelligible with an appropriate constant name